Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -111,9 +111,15 @@ private extension ReceiptRenderer {
     private func formattedAmount() -> String {
         // We should use CurrencyFormatter instead for consistency
         let formatter = NumberFormatter()
-        formatter.minimumFractionDigits = 2
-        formatter.maximumFractionDigits = 2
-        return formatter.string(for: parameters.amount / 100) ?? ""
+
+        let fractionDigits = 2 // TODO - support non cent currencies like JPY
```

Comment:
Would you mind linking to #3948 here please?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/4441/commits/47446b1ac08a8570c149545b45214b2d9da7599c